packages/mux-player/src/helpers.ts

```diff
@@ -62,6 +63,39 @@ export function getCcSubTracks(el: MuxPlayerElement) {
   return Array.from(el.video?.textTracks ?? []).filter(({ kind }) => kind === 'subtitles' || kind === 'captions');
 }
 
+export const getLiveTime = (el: MuxPlayerElement) => {
+  const { video } = el;
+  return video?.hls?.liveSyncPosition ?? video?.seekable.length
+    ? video?.seekable.end(video.seekable.length - 1)
+    : undefined;
+};
+
+export const seekToLive = (el: MuxPlayerElement) => {
+  const liveTime = getLiveTime(el);
+  if (liveTime == undefined) {
+    console.warn('attempting to seek to live but cannot determine live edge time!');
+    return;
+  }
+  el.currentTime = liveTime;
+};
+
+export const isInLiveWindow = (el: MuxPlayerElement) => {
+  const { streamType } = el;
+  const liveTime = getLiveTime(el);
+  const currentTime = el.video?.currentTime;
+  if (liveTime == undefined || currentTime == undefined) {
+    return false;
+  }
+  const delta = liveTime - currentTime;
+  if (streamType === StreamTypes.LL_LIVE) {
+    return delta <= 1 * 3.5;
```

Comment:
lmk if some code comments would be good here. This is based on holdbacks, current mux segment vs. part sizes, and "fudge factor" due to RTT and related.

Reply:
I like having a constant for things like that. Makes it extra obvious what's happening here.